park-dashboard/src/Components/StandardParkInfo/StandardParkInfo.css

```diff
@@ -0,0 +1,79 @@
+.info-description{
+    width: 80%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 2rem;
+    background-color: #f1f1f1;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.2);
+    font-size: 16px;
+}
+.more-info {
+    display: flex;
+    justify-content: center;
+}
+.info-hours {
+    width: 30%;
+    margin-top: 2rem;
+    text-transform: capitalize;
+    background-color: #c2e9c2;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.2);
```

Comment:
You can try here -- background-color: #effbbd;
    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.2);

Reply:
Added.